src/coreclr/jit/optimizer.cpp

```diff
@@ -5319,6 +5319,15 @@ void Compiler::optCloneLoop(unsigned loopInd, LoopCloneContext* context)
         // checked them to guarantee they are clonable.
         bool cloneOk = BasicBlock::CloneBlockState(this, newBlk, blk);
         noway_assert(cloneOk);
+
+#if FEATURE_LOOP_ALIGN
+        if (blk->isLoopAlign())
+        {
+            newBlk->bbFlags &= ~BBF_LOOP_ALIGN;
```

Comment:
It would be useful here to have a comment describing _why_ we don't want the cloned loop to be have the aligned bit.